app/src/org/commcare/android/nfc/NfcReadActivity.java

```diff
@@ -104,6 +107,14 @@ private void readFromNfcTag(Tag tag) {
         }
     }
 
+    private String decryptValue(String message) throws EncryptionUtils.EncryptionException {
+        if (encryptionKey != null && message.startsWith(NFC_ENCRYPTION_SCHEME)) {
```

Comment:
I imagine this is fine as long as there's no chance that these prefix based tags might conflict with message contents.  

It might be worth thinking through whether there are circumstances where a prefix won't be enough later (for instance if we needed to write another key into the message, like a salt), but I can't quite come up with something that doesn't feel like over-solving the current problem